DATA/production/configurations/2022/LHC22f/apass1/setenv_extra.sh

```diff
@@ -233,6 +233,9 @@ if [[ $ADD_CALIB == "1" ]]; then
   fi
 fi
 
+# ad-hoc settings for TPC residual extraction
+export ARGS_EXTRA_PROCESS_o2_calibration_residual_aggregator="--output-type trackParams,unbinnedResid,binnedResid"
```

Comment:
Should we put this in the `if` at line 225?

Reply:
I think it does not hurt to have it here. In case the `o2-calibration-residual-aggregator` is not included in the global workflow, then this variable is simply ignored

Reply:
I know, but I would prefer to have it next to the relevant piece of code, I find it less confusing.

Reply:
Ok sure, I can move it. And is this setting needed for both LHC22f/apass1 and LHC22f/apass1_TPCcalib or do we anyhow only use one of the two now?

Reply:
Now we will use only the one in 22f/apass1. 
The other script is dedicated to TPC calib, so there I did not mind to not have a check. But as I said, 22f/apass1 should from now on be the default (@catalinristea , just to be sure that he uses this one from now on).
I will merge now.

Reply:
Ok, thanks a lot